tests/cephfs/cephfs_utilsV1.py

```diff
@@ -2696,3 +2843,66 @@ def _setup_crefi(node):
 
         # Setup Crefi pre-requisites : pyxattr
         node.exec_command(sudo=True, cmd="pip3 install pyxattr", long_running=True)
+
+    def generate_all_combinations(
+        self, client, ioengine, mount_dir, workloads, sizes, iodepth_values, numjobs
+    ):
+        """
+        Generates all the combimations of workloads sizes, iodepth as per the fio_config
+        Sample fio_config:
+        fio_config:
+          global_params:
+            ioengine: libaio
+            direct: "1"
+            size: ["1G"]
+            time_based: ""
+            runtime: "60"
+          workload_params:
+            random_rw:
+              rw: ["read","write"]
+              rwmixread: "70"
+              bs: 8k
```

Comment:
As discussed in standup, lets try to keep the param 'bs' also configurable for multiple values. Not a priority but an enhancement to keep a track.